src/styles/index.scss

```diff
@@ -1,3 +1,21 @@
-body {
+@import 'blocks/var';
+@import 'blocks/catalog';
+@import 'blocks/header';
+@import 'blocks/stars';
+@import 'blocks/card';
+@import 'blocks/logo';
+@import 'blocks/nav';
+
+* {
```

Comment:
it is not a good practice to use * selector for any thing but border-box property. You can reset margins and padding for particular elements. for instance:

```scss
p,
h1,
ul {
  margin: 0;
  padding: 0;
}
```